src/providers/baidu/Baidu.js

```diff
@@ -121,5 +121,5 @@ class Baidu extends Map {
     }
 }
 
-window.Map = Baidu;
+window.BaiduMap = Baidu;
```

Comment:
I'm pretty sure that could break everything if website using the library use it ? Are we sure that we want to do this ?

Reply:
Yes it's a major version :)

Reply:
Ok don't forget to explain the breaking change in the release note ;)